docker/django/Dockerfile

```diff
@@ -111,14 +111,13 @@ CMD python /opt/courtlistener/manage.py migrate && \
 #freelawproject/courtlistener:latest-web-prod
 FROM python-base as web-prod
 
-CMD gunicorn cl_wsgi:application \
-    --chdir /opt/courtlistener/docker/django/wsgi-configs/ \
+CMD gunicorn cl_asgi:application \
+    --chdir /opt/courtlistener/docker/django/asgi-configs/ \
```

Comment:
This is what loads `cl_asgi.py` which is the gunicorn config that is external from the project namespace like `cl_wsgi.py`.